site/static/css/rdf4j.css

```diff
@@ -79,3 +83,17 @@ pre code {
 pre {
   word-wrap: unset !important;
 }
```

Comment:
This was part of the css changes i did to stop the code examples from wrapping, and instead scrolling horizontally. Does that still work now?

Reply:
Yes